tests/middleware/views.py

```diff
@@ -1,6 +1,11 @@
+import sys
+
 from django.http import HttpResponse
+from django.middleware import csp
```

Comment:
Are we sure about this usage, or perhaps where this code lives? It feels a bit strange to be asking folks to import a middleware in order to use one of its constants in defining a policy. 🤔

Reply:
I personally agree having to do two different import lines `middleware.csp` and `views.decorators.csp` would be super annoying.

But at the same time, I'd probably just use 'self', 'unsafe-inline', 'none', without importing the constants, but maybe that's somehow doing it wrong?

Reply:
I mentioned in another comment that I was originally looking for a general place where Django has constants, something like `from django.constants import csp` but didn't see anything like that. Happy to move those wherever it makes the most sense if there's a suggestion?

Reply:
Just wanted to point out that the constants can help typos as well as mistakenly missing the quoting of the headers like this:

```python
SECURE_CSP = {
    "DIRECTIVES": {
        "default-src": ["self"]
        # vs the correct ["'self'"]
    }
}
```

This is incorrect and would lead to a header of `default-src self` as opposed to `default-src 'self'`.

The constants make it a little easier to avoid that.

Reply:
Ohh right excellent point. So weird that CSP domain names aren't quoted but 'self' is quoted.

Reply:
> Are we sure about this usage, or perhaps where this code lives? It feels a bit strange to be asking folks to import a middleware in order to use one of its constants in defining a policy. 🤔

I moved this to `django/conf/csp.py` since it is about configuration. There are a few other constant-like things in `django/conf/__init__.py` so this made sense to me.

Now we can do `from django.conf import csp` to access `csp.SELF`, `csp.NONCE`, etc.

Reply:
@sarahboyce I just wanted to share this thread since I originally had the constants in `django/middleware/csp.py` and then moved them to `django/conf/csp.py`. 

Having them live in `django/middleware/constants.py` seems okay but does Carlton's original comment above still feel strange if we have `from django.middleware.constants import ... as csp_constants` -- importing from a middleware path into a view?

Reply:
Perhaps `django/utils/csp.py`?